lms/templates/design-templates/live-blocks/course-listing/_course-listing-04.html

```diff
@@ -23,7 +23,13 @@ <h1 class="a--course-listing-01__section-heading">
                   <p class="a--course-tile-01__title">${a.title}</p>
                 </div>
                 <div>
-                  <p class="a--course-tile-01__organization">${a.course_weeks_duration}</p>
+                  <p class="a--course-tile-01__organization">
+                    % if a.course_duration:
```

Comment:
@johnbaldwin This related to the edx-platform PR, the course duration is in a weird situation, because in edx.org, you can set start and end dates for a course, and also set a "duration" field. We are calculating the duration in weeks for a course, but in some cases, this is not the same value that the duration field, but the edx.org API isn't releasing the duration field, so we're enabling the ability to enter a custom duration string. If the custom duration is set, we display it as is, if not we display the duration in weeks.